src/components/FeatureSetsPanel/FeatureSetsPanelDataSource/featureSetsPanelDataSource.util.js

```diff
@@ -1,4 +1,85 @@
+import { MLRUN_STORAGE_INPUT_PATH_SCHEME } from '../../../constants'
+
 export const kindOptions = [
   { label: 'CSV', id: 'csv' },
   { label: 'PARQUET', id: 'parquet' }
 ]
+
+export const comboboxSelectList = [
+  {
+    className: 'path-type-store',
+    label: 'MLRun store',
+    id: 'store://'
+  },
+  {
+    className: 'path-type-v3io',
+    label: 'V3IO',
+    id: 'v3io://'
+  },
+  {
+    className: 'path-type-s3',
+    label: 'S3',
+    id: 's3://'
+  },
+  {
+    className: 'path-type-az',
+    label: 'Azure storage',
+    id: 'az://'
+  }
+]
+
+export const generateComboboxMatchesList = (
+  url,
+  artifacts,
+  projects,
+  projectName,
+  urlProjectPathEntered,
+  urlArtifactPathEntered,
+  urlArtifactReferencePathEntered,
+  artifactsReferences,
+  urlProjectItemTypeEntered
+) => {
+  if (!urlProjectItemTypeEntered) {
+    if (
+      projectItemsPathTypes.find(type =>
```

Comment:
If you want to have a boolean here you could use `Array.prototype.some` instead:
```suggestion
      projectItemsPathTypes.some(type =>
```